move2kube/application.properties

```diff
@@ -7,7 +7,10 @@ mp.messaging.incoming.kogito_incoming_stream.method=POST
 # This property is used when sending the notification while waiting for Q&A
 move2kube_url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
 move2kube_get_plan_max_retries=${MOVE2KUBE_GET_PLAN_MAX_RETRIES:10}
+
 # This property is used to send requests to the move2kube instance
 quarkus.rest-client.move2kube_yaml.url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
+
 # This property is used to send requests to the backstage notification plugin
-quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator/api/notifications/}
\ No newline at end of file
+quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator}
+quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN}
```

Comment:
Please set a default value, otherwise you will have an error in the e2e:
```
2024-07-10T12:14:41.487270118Z stderr F Caused by: java.util.NoSuchElementException: SRCFG00011: Could not expand value NOTIFICATIONS_BEARER_TOKEN in property quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token
```

`quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN:-}`